static/app/views/settings/account/accountClose.tsx

```diff
@@ -147,16 +150,17 @@ function AccountClose() {
       </Alert>
 
       <Panel>
-        <PanelHeader>{t('Remove the following organizations')}</PanelHeader>
+        <PanelHeader>{t('Delete the following organizations')}</PanelHeader>
         <PanelBody>
-          <PanelAlert type="info">
+          <PanelAlert type="warning">
+            <strong>{t('ORGANIZATIONS WITH CHECKED BOXES WILL BE DELETED!')}</strong>
```

Comment:
typically we prefer not to use all caps/exclamation points in in-app messaging, this might be one of the exceptions since it is irreversible

Reply:
I'm in inclined to keep, just because in the original issue, the guy (feedback giver) said them text regarding organization deletion was too small/missable.